packages/@guardian/source-foundations/src/colour/palette.stories.tsx

```diff
@@ -0,0 +1,184 @@
+import type { Story } from '../../../../../lib/@types/storybook-emotion-10-fixes';
```

Comment:
This file icon being different to the other docs looks a bit odd in the menu: 
![image](https://user-images.githubusercontent.com/15648334/180839876-d374f410-43ed-45f4-9d56-4cc71c9a86cd.png)
Also, we have lost the heading and example documentation from the top of the file: 
<img width="1067" alt="image" src="https://user-images.githubusercontent.com/15648334/180840123-f2502be5-5a59-491c-b7d8-9a5a08126daf.png">
Should we maybe separate the doc and the palette so that users can still view the documentation, and chromatic can happily test the palette colours?

Reply:
This one is complex for a number of reasons. Let's discuss and report back actions here.

Reply:
Discussed in person, outcome was use `mdx` rather than `tsx` story which resolves above points, apart from the different icon. We think though it makes sense to add chromatic examples using the `<Story />` tag for other similar `source-foundations`.

Reply:
I'm still not getting the 'docs' view by default when I run locally - so no heading or example. I also don't see any styling on the sidebar. I wonder if there's a way to 'force' the docs view? 

![image](https://user-images.githubusercontent.com/15648334/181019552-80ea7b20-a492-4c00-828a-b5f35141adac.png)

Reply:
Update: I tried upgrading storybook to latest but that didn't fix the issue.

Reply:
that's annoying - could you filter out props that begin with `__`? and then maybe add a comment explaining why so we can remove at some point when it's fixed we wonder why we're doing something so strange?

Reply:
i guess this is because we're doing something quite weird with the story

Reply:
Yeah I think that makes sense. I'd been wondering if there was a way to add a type check but this is probably simpler esp if it's a know bug.

Reply:
I was able to resolve this by specifying that we use `react-docgen` rather then `react-docgen-typescript` in `main.js`
https://github.com/storybookjs/storybook/issues/9832